src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -121,7 +121,6 @@ class PolicyHandler : public PolicyHandlerInterface,
   void GetUpdateUrls(const uint32_t service_type,
                      EndpointUrls& out_end_points) OVERRIDE;
   virtual std::string GetLockScreenIconUrl() const OVERRIDE;
-  void ResetRetrySequence() OVERRIDE;
```

Comment:
@jacobkeeler this method is unused. Remiving this method extracted to separate commit 0682954